content/en/dashboards/ddsql_editor/reference/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: DDSQL Reference
+---
+
+{{< callout url="https://google.com">}}
+DDSQL is in private beta.
+{{< /callout >}}
+
+## Overview
+
+DDSQL is a query language for Datadog data. It implements several standard SQL functions, such as `SELECT`, along with new functions for working with Datadog tags, metrics, and so on. For example, [the `AGGR` statement][5], which aggregates metrics data, is unique to DDSQL.
+
+You can use DDSQL to query your infrastructure in the [DDSQL Editor][2] in Datadog.
+
+DDSQL uses a [schema on read](#schema-on-read) approach to allow queries against unstructured data, such as [tags][1]. Each query is executed within a [session](#sessions) that can be configured with environment variables.
+
+## Supported SQL syntax
```

Comment:
this section seems too pedantic and focusing on irrelevant details for the scope of this doc.

Reply:
@cangencer I agree that the presentation feels academic, I just wasn't sure of a workaround that wouldn't make the other docs more confusing. Some function signatures / query templates use placeholders like `DQL_expression` or `DQLExpr`, which link to this section. I do think the user will have the question "what areas of SQL are supported?", a question that feels in scope for the page and is answered by this table. So I opted for clarity and just having it in one place. But suggestions are welcome on how to improve it, since it's a little awkward.

Reply:
I would just skip it but also don't have a strong opinion, I don't think it adds much value in itself - as most of the syntax is covered elsewhere. If you think users would find it still valuable to understand what major parts of SQL are supported, we can leave it in.

Reply:
Mainly I don't want to use terms like "DQL expression" without including it, so I'll leave it for now. But I'll ask the PM whether there's a more elegant approach. Thank you!